usmqe_tests/api/tendrl/others/test_user.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf8 -*-
+"""
+REST API test suite - user
+"""
+
+import json
+import pytest
+from usmqe.api.tendrlapi import user as tendrlapi_user
+from usmqe.api.tendrlapi.common import login, logout
+
+
+LOGGER = pytest.get_logger('user_test', module=True)
+"""@pylatest default
+Setup
+=====
+
+Prepare USM cluster accordingly to documentation.
+
+Further mentioned ``APIURL`` points to: ``http://USMSERVER/api/1.0``.
+"""
+
+"""@pylatest default
+Teardown
+========
+"""
+
+
+def test_user_get(valid_session_credentials):
```

Comment:
Because a fixture for creating and removing a user was added, I think this test should use it and call proper `check_user()`.
However admin authentication could be left as it is, so the test uses both `valid_new_user` and `valid_session_credentials`.